src/components/DateRangeFIlter/DateRangeFilter.tsx

```diff
@@ -15,11 +15,16 @@ import { RadioGroup, RadioGroupItem } from '@/components/ui/radio-group.tsx';
 import { cn } from '@/lib/utils.ts';
 import { Route } from '@/routes/(admin)/_admin.access-keys.tsx';
 
-export const DateRangeFilter = () => {
+export interface DateRangeFilterProps {
+  withRevokedRange?: boolean;
+}
+
+export const DateRangeFilter = ({ withRevokedRange }: DateRangeFilterProps) => {
   const [dateRangeOption, setDateRangeOption] = useState<string>('createdRange');
+  const currentDate = new Date();
   const [date, setDate] = React.useState<DateRange | undefined>({
-    from: subDays(new Date(), 20),
-    to: new Date(),
+    from: subDays(currentDate, 20),
+    to: currentDate,
```

Comment:
Maybe it would look better if you move this to a function above:
```typescript
const initialTimeRange = () => {
   const currentDate = new Date();
   return {
        from: subDays(currentDate, 20),
        to: currentDate,
   }
}

//and inside the func component:
const [date, setDate] = React.useState<DateRange | undefined>(initialTimeRange())
```

Reply:
Moved a function.